pkg/manifest/ostree_deployment.go

```diff
@@ -324,6 +326,15 @@ func (p *OSTreeDeployment) serialize() osbuild.Pipeline {
 		pipeline.AddStage(systemdStage)
 	}
 
+	if len(p.EnabledPresets) != 0 || len(p.DisabledPresets) != 0 {
```

Comment:
@supakeen so changes here is meaningless, I understood it as-  where it presets added to pipeline.